<a href="https://colab.research.google.com/github/lm8201/DeepLearningWeek1/blob/main/DeepLearningWeek1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Deep Learning Week 1, LeeAnn Markwalter

In [1]:
import pandas as pd

In [16]:
df = pd.read_csv('life_expectancies.csv')

In [17]:
df

Year       Race  ... Average Life Expectancy (Years)  Age-adjusted Death Rate
0     1900  All Races  ...                            47.3                   2518.0
1     1901  All Races  ...                            49.1                   2473.1
2     1902  All Races  ...                            51.5                   2301.3
3     1903  All Races  ...                            50.5                   2379.0
4     1904  All Races  ...                            47.6                   2502.5
...    ...        ...  ...                             ...                      ...
1066  2014      White  ...                            76.7                    853.4
1067  2015      White  ...                            76.6                    861.9
1068  2016      White  ...                            76.4                    858.9
1069  2017      White  ...                            76.4                    863.4
1070  2018      White  ...                             NaN                    853.1

[1071 rows x 5 columns]

In [18]:
df.describe()

Year  Average Life Expectancy (Years)  Age-adjusted Death Rate
count  1071.000000                      1065.000000              1071.000000
mean   1959.000000                        64.500188              1593.061625
std      34.367176                        11.843765               682.369379
min    1900.000000                        29.100000               611.300000
25%    1929.000000                        57.100000              1012.950000
50%    1959.000000                        66.800000              1513.700000
75%    1989.000000                        73.900000              2057.150000
max    2018.000000                        81.400000              3845.700000

The summary statistics for the Average Life Expectancy (Years) variable provides valuable insights about the distrubtion of the dataset. First, there is a signifant difference between the maximum average life expectancy (81.4 years) and minimum average life expectancy (29.1 years). This highlights the disparities of health based on gender, time period, and race. Furthermore, the mean summary statistic tells us that the average life expectancy across all times, races, and genders is 64.5 years. 

I do not think that additional processing of the dataset is needed to improve relaibility of the summary statistics. This is because the summary statistics as displayed are useful for anlaysis and make sense in the context of the information that the dataset is providing. However, in future analysis it would be informative to obtain the summary statistics for more speicific variables. For instance, it would be interting to compare the average life expectancies and death rates overtime based on gender and rate. 